diagnostics/ENSO_MSE/input_data/obs_data/ENSO_MSE/enso_mse

```diff
@@ -0,0 +1 @@
+
```

Comment:
is this file missing text? If not, delete

Reply:
deleted